### PR TITLE
Format python files to PEP8

### PR DESCRIPTION
**Pull request by bbondy**
Fix https://github.com/brave/brave-browser/issues/2081

### CODE REVIEW

File: script/test/test_upload.py [hunk 1]
```diff
@@ -7,117 +7,158 @@
...
+                                 'BraveBrowserStandaloneSetup.exe'])
+
+    def test_only_returns_release_win_ia32_package(self):
+        upload.PLATFORM = 'win32'
+        os.environ['TARGET_ARCH'] = 'ia32'
+        pkgs = list(upload.get_brave_packages(
+            os.path.join(self.get_pkgs_dir, upload.PLATFORM),
+            'release', '0.56.8'))
+        self.assertEquals(pkgs, ['BraveBrowserStandaloneSetup32.exe',
+                                 'BraveBrowserSetup32.exe'])
```
Comment (srirambv) [inline script/test/test_upload.py]: Needs another blank line here

File: script/test/test_publish.py [hunk 2]
```diff
@@ -15,24 +16,24 @@
...
+        self.repo.releases._releases = [{'tag_name': 'old', 'draft': False}]
+        self.assertRaises(UserWarning, publish_release.get_draft, self.repo,
+                          'new')
 
-  def test_succeeds_on_existing_draft(self):
-    self.repo.releases._releases = [{'tag_name': 'test', 'draft': True}]
-    publish_release.get_draft(self.repo, 'test')
+    def test_succeeds_on_existing_draft(self):
+        self.repo.releases._releases = [{'tag_name': 'test', 'draft': True}]
+        publish_release.get_draft(self.repo, 'test')
```
Comment (srirambv) [inline script/test/test_publish.py]: Needs another blank line here

File: script/test/test_helpers.py [hunk 3]
```diff
@@ -8,72 +8,76 @@
...
+        )
+        self.assertEqual(self.retry_func.ran, 1)
+
+    def test_no_run_catch_func_on_success(self):
+        retry_func(
+            self.retry_func.succeed,
+            catch_func=self.catch_func.succeed,
+            catch=UserWarning, retries=3
+        )
+        self.assertEqual(self.catch_func.ran, 0)
```
Comment (srirambv) [inline script/test/test_helpers.py]: Needs another blank line here

File: script/lib/config.py [hunk 4]
```diff
@@ -8,112 +8,129 @@
...
 def get_chromedriver_version():
-  pattern = "^chromedriver_version = \"([0-9]\.[0-9]+)\""
-  build_gn_path = os.path.join(BRAVE_CORE_ROOT, 'BUILD.gn')
-  with open(build_gn_path, 'r') as build_gn_file:
-    for line in build_gn_file:
-      match = re.search(pattern,line)
-      if match:
-        version = match.group(1)
-  return 'v' + version
+    pattern = "^chromedriver_version = \"([0-9]\.[0-9]+)\""
```
Comment (srirambv) [inline script/lib/config.py]: W605 invalid escape sequence '\.'

**Comment by bbondy** [comment]
master: bebcb4e26321da7aa370b182b8cd508d525eca98
0.58.x: a5d4b52e893f7784676adffb25d572a2f09f7d95
0.57.x: 04e5c905b7e608acb7505356bdfa6c2da5c62eca

**Comment by bbondy** [comment]
uplifted since it's formatting only and it'll make other merges easier for l10n.

**Pull request merged**